platform/cog-platform-fdo.c

```diff
@@ -54,6 +63,26 @@
 # define HAVE_DEVICE_SCALING 0
 #endif /* WPE_CHECK_VERSION */
 
+#if COG_ENABLE_WESTON_DIRECT_DISPLAY
+#define VIDEO_BUFFER_FORMAT DRM_FORMAT_YUYV
+struct buffer {
```

Comment:
I think this struct would be better named `video_buffer`, given it's used along with `video_surface`.